src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -1078,7 +1078,7 @@ public void sendOneWay(MessageOut message, int id, InetAddressAndPort to)
             logger.trace("{} sending {} to {}@{}", FBUtilities.getBroadcastAddressAndPort(), message.verb, id, to);
 
         if (to.equals(FBUtilities.getBroadcastAddressAndPort()))
-            logger.trace("Message-to-self {} going over MessagingService", message);
+            logger.debug("Message-to-self {} going over MessagingService", message);
```

Comment:
This really needs to be NoSpamLogger? How many times a second might we do this in some cases? What if new code comes that causes this to occur many times a second?

In OSS debug logging is on all the time. That's really not a risk we should be taking.

Reply:
To be honest, I think relying on the logging might have been a bad idea. I've left this statement with `trace` and have rewritten the tests instead.